homeassistant/components/jewish_calendar/const.py

```diff
@@ -10,11 +10,22 @@
 CONF_DIASPORA = "diaspora"
 CONF_CANDLE_LIGHT_MINUTES = "candle_lighting_minutes_before_sunset"
 CONF_HAVDALAH_OFFSET_MINUTES = "havdalah_minutes_after_sunset"
+CONF_CALENDAR_EVENTS = "calendar_events"
 
 DEFAULT_NAME = "Jewish Calendar"
 DEFAULT_CANDLE_LIGHT = 18
 DEFAULT_DIASPORA = False
 DEFAULT_HAVDALAH_OFFSET_MINUTES = 0
 DEFAULT_LANGUAGE = "en"
+DEFAULT_CALENDAR_EVENTS = [
+    "date",
+    "holiday",
+    "weekly_portion",
+    "omer_count",
+    "daf_yomi",
```

Comment:
I'd suggest to move these and `candle_lighting` + `havdalah` below be to a string enum

Reply:
Done